_posts/2022-05-16-opentelemetry-metrics-for-data-prepper.md

```diff
@@ -94,14 +94,16 @@ You just need to configure the source and processor plugins:
 ```yaml
 metrics-pipeline:
   source:
-    otel_metrics_source::
+    otel_metrics_source:
+      ssl: false
```

Comment:
@KarstenSchnitter So generally we don't allow disabling of SSL in the demos. I know it can be frustrating but many times these demos get rolled into people's production so we try to be secure first. I am going to get a second opinion though. 

@dlvenable - What are your thoughts?

Reply:
We are currently disabling SSL in most of our examples. That is driven by a desire to make them easier to run. We are making changes in Data Prepper 2.0 to support more SSL-by-default scenarios, and can probably start to make more demos use SSL.

This is a blog post, not a demo project. Since this configuration needs to be copied and pasted somewhere, I think it is even more appropriate to keep SSL active. So lean toward just dropping the `ssl: false` here, even if we temporarily have it in demo projects.

Reply:
```suggestion
```